src/MenuItem.js

```diff
@@ -10,13 +10,15 @@ const MenuItem = React.createClass({
     target:    React.PropTypes.string,
     onSelect:  React.PropTypes.func,
     eventKey:  React.PropTypes.any,
-    active:    React.PropTypes.bool
+    active:    React.PropTypes.bool,
+    tabIndex:  React.PropTypes.string
   },
 
   getDefaultProps() {
     return {
       href: '#',
-      active: false
+      active: false,
+      tabIndex: null
```

Comment:
This would be a breaking change since it is currently `-1` and not null. I've been planning to keep this as a `-1` in the revisited menu item to comply with keyboard accessibility as outlined in the [W3C WAI Aria guidelines](http://www.w3.org/WAI/PF/aria-practices/#menubutton) Is this something that you would still like configurable once I'm done with my re-write? #526